site/docs/09-math/07-vector.mdx

```diff
@@ -61,3 +61,169 @@ console.log(anotherPoint.toString()) // "(50, 50)"
 ```
 
 Notice how both `point` and `samePoint` share the same vector reference, so using `setTo` mutates the vector. Use `clone` to ensure you are not changing vectors unexpectedly!
+
+### Static Methods Available
+
+#### `Vector.fromAngle(angle: number): Vector`
+Creates a unit vector from an angle in radians.
+
+```typescript
```

Comment:
Would you mind using Shiki for these snippets? I'd love for all the snippets in the docs to be migrated to Twoslash snippets for type definitions and so they get compiled/type-checked in CI. You can reference the Events or Actors docs for how to set it up.